src/remote-control/remote-control-service.js

```diff
@@ -201,6 +218,56 @@ class RemoteControlService {
             this._getSpotId(), COMMANDS.SET_VIDEO_MUTE, { mute });
     }
 
+    /**
+     * Begins the process for establishing a connection to the meeting in order
+     * to share a local screen to a remote spot.
+     *
+     * @param {boolean} enable - Whether to start ot stop screensharing.
+     * @returns {Promise}
+     */
+    setWirelessScreensharing(enable) {
+        if (enable) {
+            const connection = new ScreenshareService({
+
+                /**
+                 * Callback invoked when the connection has been closed
+                 * automatically. Triggers cleanup of
+                 * {@code ScreenshareService}.
+                 *
+                 * @returns {void}
+                 */
+                onConnectionClosed: () => this._delegate.stopScreenshare(),
```

Comment:
i don't have a full understanding of the role of the delegate/what's in there vs what's in this screenshare service.  maybe you can explain it to me more.